src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -139,9 +125,12 @@ public static void main(final String[] args) {
       usage();
       return;
     }
-    SwingUtilities.invokeLater(LookAndFeel::setupLookAndFeel);
 
-    HttpProxy.setupProxies();
+    LookAndFeel.setupLookAndFeel();
```

Comment:
> I do believe it needs to be called before the Swing components are created.

Right.  The unfortunate thing about this `main()` method is it's dealing with two orthogonal UI systems: Swing and JFX.  Regardless of where this call has to be to make the Swing client work correctly, my only concern is that it might have to be wrapped in a `!useJavaFxUi` guard.  Then we would have three such guards in this method. 😞